src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.mocks.ts

```diff
@@ -59,6 +59,10 @@ export const season1Badges: CollectionBadge[] = [
   {
     variant: 'dryIce',
   },
+  {
+    variant: 'mellowLpVault',
+    achievedAt: new Date('02/02/2022').valueOf(),
```

Comment:
what if the user didn't achieve this? where do we verify this against subgraph data?

Reply:
this is storybook. Not used anywhere in Application. This is only for the purpose of demo. check the mappers in getSeasonBadges for the actual logic

Reply:
ok, got it